packages/styleguide/src/components/Marker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { color as d3Color } from 'd3-color'
+
+export const MarkerConfig = {
+  yellow: {
+    color: 'rgb(255,255,0)',
+  },
+  pink: {
+    color: 'rgb(255,100,255)',
+  },
+  green: {
+    color: 'rgb(0,255,0)',
+  },
+  blue: {
+    color: 'rgb(0,230,230)',
+  },
+  rotstift: {
+    style: () => ({
+      borderBottom: `3px solid`,
+      borderBottomColor: 'red',
+    }),
+    pickerInnerStyle: (isSelected) => ({
+      borderRadius: 4,
+      position: 'absolute',
+      backgroundColor: 'red',
+      bottom: 0,
+      left: 0,
+      right: 0,
+      height: 3,
+      boxShadow: isSelected && 'red 0px 0px 2px',
+    }),
+  },
+  drop: {
+    style: (isSelected) => ({ opacity: isSelected ? 0.6 : 0.3 }),
+    pickerColor: 'rgba(0,0,0,0.1)',
+  },
+}
+
+export const markerKeys = Object.keys(MarkerConfig)
+
+export const Marker: React.FC<{
+  marker: string
```

Comment:
Any specific reason why this isn't typed? from what I see there is a limited choice of colors.